examples/analytics.amp.html

```diff
@@ -127,6 +127,22 @@
 </script>
 </amp-analytics>
 
+<!-- Krux tracking. Ensure you replace CONFID with your configuration ID. -->
+<amp-analytics type="krux" config="https://cdn.krxd.net/controltag/amp/CONFID.json">
```

Comment:
As noted in the comment above. The actual URL will vary by publisher so they should replace CONFID with the actual configuration ID for their account. Is there a clearer way I should say this here?